src/actions.js

```diff
@@ -29,21 +29,16 @@ function _pageAndEdges(projections) {
 }
 
 export function filter_location_by_parents(district_uuid, region_uuid, filters, location_type) {
-  let parentFilter = '';
+  let parentFilter = "";
   if (district_uuid) {
-    if (location_type == 'W')
-      parentFilter = `parent_Uuid: "${district_uuid}"`
-    else if (location_type == 'V')
-      parentFilter = `parent_Parent_Uuid: "${district_uuid}"`
-  }
-  else if (region_uuid) {
-    if (location_type == 'W')
-      parentFilter = `parent_Parent_Uuid: "${region_uuid}"`
-    else if (location_type == 'V')
-      parentFilter = `parent_Parent_Parent_Uuid: "${region_uuid}"`
+    if (location_type == "W") parentFilter = `parent_Uuid: "${district_uuid}"`;
```

Comment:
Is this possible to create the const for 'W' and 'V' to have a better view what does mean exactly? (I can assume that W is ward and V means village etc).
It could be good to have such const to assign such value in particular places like condition etc.

Reply:
ok, noted, therefore we can skip changing this, @olewandowski1